samples/rust/Kconfig

```diff
@@ -110,4 +110,14 @@ config SAMPLE_RUST_RANDOM
 
 	  If unsure, say N.
 
+config SAMPLE_RUST_SEQ_FILE
```

Comment:
This may need a dependency on `PROC` etc.

Reply:
In other words, if we build the sample but the kernel does not have `/proc` support, it should not fail the build.

Reply:
done